.github/workflows/release.yml

```
@@ -0,0 +1,230 @@
+name: Release
+on:
+  push:
+    tags:
+      - v[0-9]+.[0-9]+.[0-9]+
+  workflow_dispatch:
+    inputs:
+      tag_name:
+        description: 'Tag name for release'
+        required: false
+        default: nightly
```

```suggestion
        default: 0.27-nightly
```
just to make sure if one does trigger it without a specific name, it doesn't collide with the `nightly` tag from `main`